merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/SerializedValue.java

```diff
@@ -239,7 +248,7 @@ protected Optional<T> onDefault() {
    *
    * @return True if this object represents a null value, and false otherwise.
    */
-  public boolean isNull() {
+  default boolean isNull() {
```

Comment:
Why are `OptionalVisitor`'s methods changed? It's still an `abstract class`, so we still want its methods explicitly `public` (and superclass methods are already `default` by, er, default).

Reply:
Ugh, yes, I thought `OptionalVisitor` was a longer class. (I missed the closing brace.)